packages/lodestar/src/sync/range/chain.ts

```diff
@@ -201,6 +201,10 @@ export class SyncChain {
     return toArr(this.batches).map((batch) => batch.getMetadata());
   }
 
+  get startEpochValue(): Epoch {
```

Comment:
Is this function used somewhere?

Reply:
nope :slightly_smiling_face: , thank you for pointing out!, removed.